packages/rancher-sriov/generated-changes/patch/templates/clusterrolebinding.yaml.patch

```diff
@@ -0,0 +1,18 @@
+--- charts-original/templates/clusterrolebinding.yaml
++++ charts/templates/clusterrolebinding.yaml
```

Comment:
Is the patch name for the `charts/templates/clusterrolebinding.yaml` file?  
It more looks like for the `charts/templates/clusterrole.yaml` file according to the contents. 
Can you double check with it?

Reply:
Good catch :). It's a bug upstream. I noticed it while creating this patch and fixed it upstream ==> https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/138, but it is not yet in the release we are using

Reply:
oh, interesting bug 😄 

We need to fix in our patch file, otherwise the `make prepare` will fail 
```
➜  charts git:(sriov-chart2) ✗ PACKAGE=rancher-sriov make prepare
./bin/charts-build-scripts prepare
INFO[0000] Pulling k8snetworkplumbingwg/sriov-network-operator@v1.0.0[path=deployment/sriov-network-operator] from upstream into charts 
INFO[0020] Loading dependencies for chart               
INFO[0020] Applying changes from generated-changes      
INFO[0020] Applying: generated-changes/patch/Chart.yaml.patch 
INFO[0020] Applying: generated-changes/patch/templates/NOTES.txt.patch 
INFO[0020] Applying: generated-changes/patch/templates/_helpers.tpl.patch 
INFO[0020] Applying: generated-changes/patch/templates/clusterrolebinding.yaml.patch 
ERRO[0020] 
patching file templates/clusterrolebinding.yaml
Hunk #1 FAILED at 15.
1 out of 1 hunk FAILED -- saving rejects to file templates/clusterrolebinding.yaml.rej 
FATA[0020] Encountered error while preparing main chart: Encountered error while trying to apply changes to charts: Unable to generate patch with error: exit status 1 
make: *** [prepare] Error 1
```